src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -473,7 +474,156 @@ pass::EliminateSqueeze::EliminateSqueeze() {
         return false;
     };
 
-    auto m = std::make_shared<pattern::Matcher>(squeeze_pattern, matcher_name);
+    auto m = make_shared<pattern::Matcher>(squeeze_pattern, matcher_name);
+    this->register_matcher(m, callback);
```

Comment:
so it wasn't registered!?

Reply:
interesting, I will double-check

Reply:
no, it was ok, you can find `this->register_matcher(m, callback);` below in EliminateSplitConcat transformation.
looks like a bug of Diff tool